### DISCUSSION TITLE
How do I change the font of my blog page title?

### DISCUSSION THREAD

**Discussion by Hawaii-Yan** [alshedivat/al-folio#2107] [Q&A] [answered]
Dear contributors, how do I change the font of my blog page title?
 I uploaded the fonts I want to use in the assets/fonts folder, but I don't know where to set the substitution. 
 I hope someone can answer me. Thank you.

**Comment by george-gca** [comment, marked as answer]
I would recommend creating it with the [CSS @font-face rule](https://www.w3schools.com/css/css3_fonts.asp) and then setting this one on the [file responsible for drawing blog's main page](https://github.com/alshedivat/al-folio/blob/master/_pages/blog.md) as an [inline css](https://www.w3schools.com/html/html_css.asp) on the right place. Just check for where in that page the title is set with `site.blog_name`.
